embedded/ActuatorBoard/readme.md

```diff
@@ -1,27 +0,0 @@
-**Actuator Board**
```

Comment:
@duggerd wants to store all of the electrical documentation and firmware on the MIL Confluence server.